### DISCUSSION TITLE
Wildcard Search - possible?

### DISCUSSION THREAD

**Discussion by iamhuman2414** [microsoft-search/pnp-modern-search#2025] [Q&A] [answered]
For the life of me, I cannot get wildcard search (?, *) to work. Any ideas? 
Is wild card search even supported?

**Comment by wobba** [comment, marked as answer]
Only prefix matching is supported with the `*` wildcard. See https://docs.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference#wildcard-operator.

**Reply by iamhuman2414** [reply]
Ahhh... now I know what I was doing wrong! Should be at least a 4 character-prefix! Thanks so much!!! Much appreciated.

**Reply by wobba** [reply]
Less characters should work as well. Like `title=a*` which return items where the title field starts with an `a`.
